### PR TITLE
feat(ci/deploy-director) - deploy director VM, stemcells to Azure

### PR DESCRIPTION
**Pull request by ritazh**

### CODE REVIEW

File: ci/tasks/azure-provision.yml [hunk 1]
```diff
@@ -15,6 +15,5 @@ params:
     AZURE_REGION_SHORT_NAME: replace-me
     AZURE_STORAGE_ACCOUNT_NAME: replace-me
     AZURE_SUBSCRIPTION_ID: replace-me
-    AZURE_TENANT_ID: replace-me
```
Comment (sedouard) [inline ci/tasks/azure-provision.yml:18]: We definitely need this one.

File: ci/tasks/azure-provision.sh [hunk 2]
```diff
@@ -45,4 +45,4 @@ cat ./azuredeploy-parameters.json
 azure group deployment create ${AZURE_GROUP_NAME} --template-file ./azuredeploy.json --parameters-file ./azuredeploy-parameters.json
 
 echo -e "export DIRECTOR=$(azure network public-ip show ${AZURE_GROUP_NAME} pubIP-bosh --json | jq '.ipAddress')" >> $export_file
-echo -e "export AZURE_STORAGE_ACCESS_KEY=$(azure storage account keys list ${AZURE_STORAGE_ACCOUNT_NAME} -g sedouard-cloudfoundry --json | jq '.storageAccountKeys.key1')" >> $export_file
\ No newline at end of file
+echo -e "export AZURE_STORAGE_ACCESS_KEY=$(azure storage account keys list ${AZURE_STORAGE_ACCOUNT_NAME} -g ${AZURE_GROUP_NAME} --json | jq '.storageAccountKeys.key1')" >> $export_file
```
Comment (sedouard) [inline ci/tasks/azure-provision.sh:48]: Thanks! :-)

**Comment by sedouard** [comment]
Amazing! :-). But put back the tenant ID for `azure-provision` as it does need that variable.

File: ci/tasks/azure-provision.yml (hunk 1 above)
Comment (ritazh) [inline ci/tasks/azure-provision.yml:18]: AZURE_TENANT_ID is already there:
https://github.com/sedouard/bosh-azure-cpi-release/blob/feat_3/ci/tasks/azure-provision.yml#L11

Comment (sedouard) [inline ci/tasks/azure-provision.yml:18]: Ah nice catch :-).

**Comment by ritazh** [comment]
close #3

**Pull request merged**